mage/query-modules/python/import-util.md

```diff
@@ -0,0 +1,199 @@
+---
+id: import-util
+title: import_util
+sidebar_label: import_util
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-import_util-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/import_util.py)
+
+
+## Abstract
+
+Module for importing data from different formats.
```

Comment:
Import format should be explained here.

Reply:
I added it. Let me know whether it's okay now.